### PR TITLE
Update pelayouts for EAM tri-grid tests

### PR DESCRIPTION
**Pull request by amametjanov**
Update pelayouts for EAM tri-grid tests running into OOM errors on 4 Anvil nodes.

[BFB]

### CODE REVIEW

**Comment by rljacob** [comment]
Before: https://my.cdash.org/viewTest.php?onlyfailed&buildid=2145037
After: https://my.cdash.org/viewTest.php?buildid=2145156

**Comment by sarich** [comment]
@rljacob I don't understand your comment.

**Comment by rljacob** [comment]
I was just moving the URL's Az provided out of the PR description.   We don't want those in the merge commit.

**Comment by sarich** [comment]
Ok I didn't like them either and edited them out

File: cime_config/allactive/config_pesall.xml [hunk 1]
```diff
@@ -1460,19 +1460,19 @@
...
+        <comment>pelayout for tri-grid tests with EAM</comment>
+        <ntasks>
+          <ntasks_atm>-8</ntasks_atm>
+          <ntasks_lnd>-8</ntasks_lnd>
+          <ntasks_rof>-8</ntasks_rof>
+          <ntasks_ice>-8</ntasks_ice>
+          <ntasks_ocn>-8</ntasks_ocn>
+          <ntasks_glc>-8</ntasks_glc>
+          <ntasks_wav>-8</ntasks_wav>
+          <ntasks_cpl>-8</ntasks_cpl>
```
Comment (jonbob) [inline cime_config/allactive/config_pesall.xml:1475]: @amametjanov - what do the negative task numbers mean? I'm having a user running into trouble on cori with this resolution

Comment (amametjanov) [reply cime_config/allactive/config_pesall.xml:1475]: Negative numbers note `node` count, positive numbers note MPI `task` count (divide by `MAX_MPITASKS_PER_NODE` to get the node count)

Comment (jonbob) [reply cime_config/allactive/config_pesall.xml:1475]: Thanks for the explanation -- I did not know that. This PR is still causing issues on cori-knl -- a trigrid test setting up -compset WCYCL1850 -res ne30pg2_r05_EC30to60E2r2 fails with this error:
````
grid_choice a%ne30np4.pg2 grid_match a%ne30np4.*_l%r05_oi%EC.+
compset_choice any compset_match any
pesize_choice any pesize_match any
points = 10
ERROR: More than one PE layout matches given PE specs
````
But that error goes away if I get rid of this change

**Review by jonbob — COMMENTED** [review]
Sorry -- I meant this to be a comment and not a review

**Pull request merged**